strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -54,6 +69,205 @@ def __init__(self):
         self._supported["mixed_states"] = True
         self._init_modes = None
         self.circuit = None
+        self.ancillae_samples_dict = {}
+
+    def run_prog(self, prog, **kwargs):
+        """Runs a strawberryfields program using the bosonic backend.
+
+        Args:
+            prog (object): sf.Program instance
+
+        Returns:
+            tuple: list of applied commands,
+                    dictionary of measurement samples,
+                    dictionary of ancilla measurement samples
```

Comment:
Or going with a bullet point structure, though worth checking the rendering.
```suggestion
            tuple: a tuple of the list of applied commands, the dictionary of measurement samples,
            and the dictionary of ancilla measurement samples
```